src/BbsBlsSignatureProof2020.ts

```diff
@@ -386,7 +394,10 @@ export class BbsBlsSignatureProof2020 extends suites.LinkedDataProof {
    * @param documentLoader {function}
    * @param expansionMap {function}
    */
-  async getVerificationMethod({ proof, documentLoader }: any): Promise<object> {
+  async getVerificationMethod({
+    proof,
+    documentLoader
+  }: any): Promise<DidDocumentPublicKey> {
```

Comment:
Hmm, I'm not sure if this keeps the function inline with the contract interface defined by JSONLD-Signatures library. @tplooker can you have a look at this when you look over this?

Reply:
Not sure I understand what you mean? We are just typing the objects return here it looks correct to me?

Reply:
One thought I had is that DidDocumentPublicKey does not define `@context` and the security v2 context here does not define publicKeyJwk so it will never be returned from the result of the JSONLD frame operation?

Reply:
Ahh just realized because DidDocumentPublicKey is an object there's not a conflict. Was thinking our suite this contract change was going to be an issue, but realized we're just providing slightly stronger guarantees about what's returned from this function with this change rather than saying "it could be any object".

I think this is fine now actually.